src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -461,7 +490,7 @@ public void execute() throws GitException, InterruptedException {
                 // we don't run a 'git clone' command but git init + git fetch
                 // this allows launchCommandWithCredentials() to pass credentials via a local gitconfig
 
-                init_().workspace(workspace.getAbsolutePath()).execute();
+                init_().workspace(workspace.getRemote()).execute();
```

Comment:
Not sure what method this is calling but probably it should take `FilePath` rather than `String`.